includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -26,9 +26,30 @@ public function setHrefs( $hrefs ) {
 	 * @return string full URL, in case of lang specific URL missing, default one is returned
 	 */
 	public function getHref( $name, $lang ) {
+		if ( !isset( $this->hrefs[ $lang ] ) ) {
+			$lang = $this->getLangFallback( $lang );
+		}
+
 		return $this->hrefs[ $lang ][ $name ] ?? $this->hrefs[ 'default' ][ $name ];
 	}
 
+	private function getLangFallback( $lang ) {
+		if ( isset( $this->hrefs[ $lang ] ) ) {
```

Comment:
Yep, my bad, I wrote the method first, though need to think of a better method name if we're always calling it. :)